egs2/fisher_callhome_spanish/asr1/local/mapping

```diff
@@ -0,0 +1 @@
+/projects/tir6/general/sdalmia/sid-espnet/egs2/fisher_callhome_spanish/asr1/data/local/fisher-callhome-corpus/mapping
```

Comment:
why absolute path?

Reply:
This gets created during data prep. It shouldn't have been in the commit. I have removed it now.